airflow/providers/amazon/aws/sensors/quicksight.py

```diff
@@ -0,0 +1,86 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from typing import TYPE_CHECKING, Optional
+
+from airflow.exceptions import AirflowException
+from airflow.providers.amazon.aws.hooks.quicksight import QuickSightHook
+from airflow.providers.amazon.aws.hooks.sts import StsHook
+from airflow.sensors.base import BaseSensorOperator
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+
+class QuickSightSensor(BaseSensorOperator):
+    """
+    Watches for the status of an Amazon QuickSight Ingestion.
+
```

Comment:
It would be great to reference the related section in the documentation.

```suggestion

.. seealso::
        For more information on how to use this sensor, take a look at the guide:
        :ref:`howto/sensor: QuickSightSensor `
```

Reply:
Done

Reply:
Done

Reply:
@vincbeck Appreciate the feedback and details.  I have implemented the requested changes and suggestions . Kindly review it . Thanks :)